packages/obojobo-document-engine/ObojoboDraft/Chunks/IFrame/render-settings.js

```diff
@@ -34,6 +34,14 @@ const getDisplayedTitle = modelState => {
 	return (modelState.src || '').replace(/^https?:\/\//, '')
 }
 
+const getAriaRegionLabel = (modelState, displayedTitle) => {
+	if (modelState.title) {
+		return 'External content titled "' + displayedTitle + '" from ' + modelState.src + '.'
```

Comment:
I think we can probably remove "titled" for conciseness without losing clarity
```suggestion
		return 'External content "' + displayedTitle + '" from ' + modelState.src + '.'
```